core.d.ts

```diff
@@ -180,7 +180,10 @@ This method can be handy to put in a stream pipeline, but it comes with a price.
 export function fileTypeStream(webStream: AnyWebReadableStream<Uint8Array>, options?: StreamOptions): Promise<AnyWebReadableByteStreamWithFileType>;
 
 export declare class FileTypeParser {
-	detectors: Iterable<Detector>;
+	/**
+	File-type detectors
+	 */
+	readonly detectors: Detector[];
```

Comment:
`readonly` will prevent editing it, which is the point of this pull request.

Reply:
And changing from iterable to array *is* a breaking change. I'm fine with it though, but just pointing it out:

> Does not break existing API.

Reply:
I believe the `readonly` makes the property `detectors` immutable, but the content of the array can be changed. So new detectors can be added, or removed.

Reply:
The reason I put the `readonly` there is to making it a bit more difficult for the user to remove the default detectors. But it also limits the user in creating a new array of detectors. For maximum flexibility, I removed the `readonly`.